internal/operator-controller/controllers/common_controller.go

```diff
@@ -27,6 +27,23 @@ import (
 	ocv1 "github.com/operator-framework/operator-controller/api/v1"
 )
 
+const (
+	// maxConditionMessageLength is the Kubernetes limit minus some buffer for safety
+	maxConditionMessageLength = 30000
```

Comment:
If the maximum length is 32768 according to kubernetes validation, we can use all of that. There's no need to have an extra bit of space for safety.

Reply:
We can use 32.000

Reply:
Addressed